src/drivers/ins/microstrain/module.yaml

```diff
@@ -6,305 +6,265 @@ parameters:
 
         MS_MODE:
             description:
-                short: Toggles using the device as the primary EKF
+                short: MicroStrain device mode
                 long: |
-                    Setting to 1 will publish data from the device to the vehicle topics (global_position, attitude, local_position, odometry), estimator_status and sensor_selection
-                    Setting to 0 will publish data from the device to the external_ins topics (global position, attitude, local position)
-                    Restart Required
-
-                    This parameter is specific to the MicroStrain driver.
-            type: int32
-            default: 1
+                    Sensor mode publishes raw IMU data to be used by EKF2. INS data from the device is published to the external INS topics.
+                    INS mode publishes the INS data to the vehicle topics to be used for navigation.
+                    Driver restart required.
```

Comment:
For all these, my understanding is that you need to use the right field to indicate restart required
```
            reboot_required: true
```

For example https://github.com/PX4/PX4-Autopilot/blob/main/src/drivers/actuators/vertiq_io/module.yaml#L248

So I'd make that change and remove the text on rebooting

Reply:
It does. But for most users they wouldn't (and shouldn't) know how to just reboot a driver. So its easier and safer just to tell them to reboot the whole thing.

Reply:
You could keep the text driver restart if you want. Let me know when you have updated.

@dakejahl Below are YAML changes. They make sense to me, but would be great to get an independent scan.